apps/i18n/common/en_us.json

```diff
@@ -517,6 +517,7 @@
   "downloadAssessmentCSV": "Download CSV of student responses",
   "downloadFeedbackCSV": "Download CSV of Feedback",
   "downloadParentLetter": "Download parent letter",
+  "downloadParentLetterTooltip": "Download parent letters for all {numStudents} students in this section",
```

Comment:
If it's just one generic letter for now, do we need to change this tooltip? Or maybe even remove it until it's relevant?

Reply:
note that you can use [MessageFormat syntax](https://messageformat.github.io/messageformat/page-guide#toc4__anchor) to make the plural work here for all cases. Something like

```
Download parent letters for {numStudents, plural, one {the student} other {all # students}} in this section
```